apps/testrunner/tasks/selenium_get.py

```diff
@@ -19,6 +19,15 @@
 logger = get_task_logger(__name__)
 
 
+class quitting(object):
```

Comment:
Yeah, I thought about putting this into common before I realized it's the only place where it is used and will probably be used.